### PR TITLE
Releasing a project infographic

### PR DESCRIPTION
**Pull request by discombobulateme**
Added an infographic style image to resume all the steps to release an open source project in Sauce Labs

### CODE REVIEW

File: content/docs/releasing.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 title: "Releasing a new Open Source project"
 description: "Releasing a new Open Source project"
 draft: false
+images:
+    - /images/docs/about-white.svg
 ---
 
 Here you can see the steps that describe how Saucers can release a new open source project. This is a simple process, and if you need help going through it the Open Source team is here to give you a hand.
 
+{{< figure src="/images/docs/about-white.svg" class="img-responsive text-center" >}}
```
Comment (christian-bromann) [inline content/docs/releasing.md]: I suggest to move the graphic to the bottom of the page. It might be more useful to sum things up with it.

**Review by christian-bromann — COMMENTED** [review]
I wonder if we should just take the bottom half of the info graphic. The first part explaining about the OSPO is not very related to the content.

**Comment by discombobulateme** [comment]
Changes: releasing image and added contributions image, without "about ospo"

File: content/docs/contributing.md [hunk 2]
```diff
@@ -33,9 +37,10 @@ The following criteria must be met when contributing to an open source project:
 Contributions to open source projects such as the following normally do not require you to sign a CLA, are encouraged as part of your employment at Sauce Labs, and do not need any approval:
 
 - Reporting issues
-- Writing documentation
+- Writing and/ or translating documentation
```
Comment (diemol) [inline content/docs/contributing.md]:
```suggestion
- Writing and/or translating documentation
```

File: content/docs/contributing.md [hunk 3]
```diff
@@ -33,9 +37,10 @@ The following criteria must be met when contributing to an open source project:
...
 - Reviewing code
-- Participating in project maintenance such as creating roadmaps, grooming backlogs, etc.
+- Participating in project maintenance such as creating roadmaps, grooming backlogs, etc
+- Helping with design needs
```
Comment (diemol) [inline content/docs/contributing.md]: This is very ambiguous, do we refer to graphic design or architectural design. I believe it is the former, so we could add a few more words about it.

File: static/images/docs/about-white.svg [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<svg width="3755" height="4985" viewBox="0 0 3755 4985" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (diemol) [inline static/images/docs/about-white.svg]: Is this file still needed?

**Review by diemol — CHANGES_REQUESTED** [review]

File: static/images/docs/about-white.svg (hunk 4 above)
Comment (discombobulateme) [reply static/images/docs/about-white.svg]: No it is not :) deleted!

File: content/docs/contributing.md (hunk 3 above)
Comment (discombobulateme) [reply content/docs/contributing.md]: Good point! Committed a new version

File: content/docs/releasing.md (hunk 1 above)
Comment (discombobulateme) [reply content/docs/releasing.md]: This image was changed for 2 different ones, specifically for contributions types and releasing line

**Review by diemol — DISMISSED** [review]

**Review by diemol — APPROVED** [review]
